pkg/util/cert/cert.go

```diff
@@ -37,9 +37,6 @@ const (
 
 // +kubebuilder:rbac:groups="",resources=secrets,verbs=get;list;watch;update
 // +kubebuilder:rbac:groups="admissionregistration.k8s.io",resources=mutatingwebhookconfigurations,verbs=get;list;watch;update
-// +kubebuilder:rbac:groups="admissionregistration.k8s.io",resources=validatingwebhookconfigurations,verbs=get;list;watch;update
```

Comment:
@varshaprasad96 this probably should not be removed, looking here: https://github.com/kubernetes-sigs/kueue/pull/3908/files#diff-d8800d9157feef2105fa59914df47a17ec8145a3242396fe47657894ac591241R40